### ISSUE TITLE
inet_ntop wants non const addr pointer in enet_address_get_host_ip in msvc

### ISSUE THREAD

**Issue by las3rlars** [zpl-c/enet#3]
```c++
    int enet_address_get_host_ip(const ENetAddress *address, char *name, size_t nameLength) {
#ifdef _MSC_VER
        if (inet_ntop(AF_INET6, (ENetAddress*)&address->host, name, nameLength) == NULL) {
#else
        if (inet_ntop(AF_INET6, &address->host, name, nameLength) == NULL) {
#endif
            return -1;
        }

        return 0;
    }

```

**Comment by inlife** [comment]
Hello @las3rlars,

Sorry, is this an error, or a suggestion? Not sure I understand, can you please elaborate more.

Thank you!

**Comment by inlife** [comment]
I think the idea behind your snippet is good. Just minor fix would be to cast to a proper type. Since ->host isn't a ENetAddress.

Also PR would be ok I think, unless you dont really wanna do it :)

And in any case, thank you!

**Comment by inlife** [comment]
was fixed in v2.1.0
